astronomer/providers/google/cloud/example_dags/example_bigquery_impersonation_key.py

```diff
@@ -0,0 +1,135 @@
+"""
+Example Airflow DAG for Google BigQuery service.
+Uses Async version of BigQueryInsertJobOperator and BigQueryCheckOperator.
+"""
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.providers.google.cloud.operators.bigquery import (
+    BigQueryCreateEmptyDatasetOperator,
+    BigQueryCreateEmptyTableOperator,
+    BigQueryDeleteDatasetOperator,
+)
+
+from astronomer.providers.google.cloud.operators.bigquery import (
+    BigQueryCheckOperatorAsync,
+    BigQueryInsertJobOperatorAsync,
+)
+
+PROJECT_ID = os.getenv("GCP_PROJECT_ID", "astronomer-airflow-providers")
+DATASET_NAME = os.getenv("GCP_BIGQUERY_DATASET_NAME", "astro_dataset")
+GCP_CONN_ID = os.getenv("GCP_CONN_ID", "google_impersonation")
+LOCATION = os.getenv("GCP_LOCATION", "us")
+EXECUTION_TIMEOUT = int(os.getenv("EXECUTION_TIMEOUT", 6))
+IMPERSONATION_KEY = os.getenv("IMPERSONATION_KEY", "")
```

Comment:
```suggestion
IMPERSONATION_CHAIN = os.getenv("IMPERSONATION_CHAIN", "")
```
in order to maintain consistency

Reply:
Fixed environment variable `IMPERSONATION_KEY` to  `IMPERSONATION_CHAIN`